### DISCUSSION TITLE
Edited isn't a link in discussions

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#44230] [Mobile] [answered]
GitHub Mobile beta

Consider this reply in a discussion:

![Screenshot_20230113-095756.png](https://user-images.githubusercontent.com/2119212/212350534-68ac61d6-65ff-4607-8d14-87ee7d72ee54.png)

In the browser I'm able to click edited to find out more information. On mobile, nothing happens.

**Comment by stevepopovich** [comment, marked as answer]
Hi @jsoref I passed this along to the team for prioritizing. Thanks!
